components/base/agilex/limo_base.go

```diff
@@ -387,6 +388,19 @@ func (base *limoBase) SetVelocity(ctx context.Context, linear, angular r3.Vector
 	_, done := base.opMgr.New(ctx)
 	defer done()
 
+	if rdkutils.Float64AlmostEqual(linear.Y, 0.0, 1) && rdkutils.Float64AlmostEqual(angular.Z, 0.0, 1) {
+		base.controller.logger.Infof("the received linear velocity %f and angular velocity %f results in a speed of 0",
+			linear, angular)
+	}
```

Comment:
will the linear and angular velocities always only care about Y and Z respectively? that's all I've seen from examples but I'm assuming there's situations in which we'd want to check the other components of the vector